crudops/main.go

```diff
@@ -0,0 +1,181 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"log"
+	"os"
+
+	"github.com/joho/godotenv"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	
+)
+
+func main() {
+	 if err := godotenv.Load(); err != nil {
+	 	log.Println("No .env file found")
+	 }
+
+	uri := os.Getenv("MONGODB_URI")
+	if uri == "" {
+		log.Fatal("Set your 'MONGODB_URI' environment variable. " +
+			"See: " +
+			"www.mongodb.com/docs/drivers/go/current/usage-examples/#environment-variable")
+	}
+	client, err := mongo.Connect(context.TODO(), options.Client().
+		ApplyURI(uri))
+	if err != nil {
+		panic(err)
+	}
```

Comment:
create a struct called MongoDB in a seperate package. have a field called mongoClient with appropriate type. whenever app starts, initialise mongo client at the beginning. if succeed, execute rest of the code, else panic